app/org/maproulette/jobs/SchedulerActor.scala

```diff
@@ -196,6 +198,63 @@ class SchedulerActor @Inject() (
     }
   }
 
+  /**
+    * This job moves expired task reviews to 'unnecessary'. To enable, set:
+    *    osm.scheduler.expireTaskReviews.interval=FiniteDuration
+    *    osm.scheduler.expireTaskReviews.olderThan=FiniteDuration
+    */
+  def expireTaskReviews(str: String): Unit = {
+    config.withFiniteDuration(Config.KEY_SCHEDULER_EXPIRE_TASK_REVIEWS_OLDER_THAN) { duration =>
+      db.withConnection { implicit c =>
+        logger.info(
+          s"Expiring task reviews older than $duration ..."
+        )
+
+        // Note in task review history we are moving these tasks to unnecessary
+        SQL(s"""INSERT INTO task_review_history
```

Comment:
The scheduler should now being running through the services and in turn the services would call the repositories for any sql. This allows us to limit any sql code in the repositories and try to limit any of it outside of those classes. So you would so something like:
```
service.updateTaskReviewHistory(...)
service.expireTaskReviews(...)
```
Or something like that, and then those service calls would call into the TaskReviewRepository, or TaskReviewHistoryRepository.

Reply:
Created a TaskReviewService and moved the sql calls into the TaskReviewRepository. Also, created an initial TaskReviewServiceSpec.